waku/node/v2/wakunode2.nim

```diff
@@ -132,6 +132,12 @@ proc start*(node: WakuNode) {.async.} =
   let wakuRelay = cast[WakuRelay](node.switch.pubSub.get())
   wakuRelay.addFilter("store", storeProto.filter())
 
+  let filterProto = WakuFilter.init()
+  node.switch.mount(filterProto)
+
+  let wakuRelay = cast[WakuRelay](node.switch.pubSub.get())
+  wakuRelay.addFilter(WakuFilterCodec, filterProto.filter())
```

Comment:
So in https://github.com/status-im/nim-waku/pull/128 I've moved the table with the filters to WakuNode. So that the same filters can be applied either for `WakuRelay` or `WakuFilter` protocol. The idea would be that you can subscribe a `WakuRelay` handler and/or a `WakuFilter` handler, which then runs the filters.notify on the same set of filters.

Does that make sense?

Reply:
Yeah @kdeme makes sense, do you have demo of usage?